docs/parameters.md

```diff
@@ -24,15 +24,15 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | "volumeInitializationRate"   | integer                                           |         |  When creating a volume from a snapshot, this parameter can be used to request a provisioned initialization rate, in MiB/s.                             |
 
 ## Restrictions
-* `gp3` is currently not supported on outposts. Outpost customers need to use a different type for their volumes.
+* `gp3` is currently not supported on First-generation outposts. First-generation outposts customers need to use a different type for their volumes.
```

Comment:
ginormous np: 

Remove that second sentence `First-generation outposts customers need to use a different type for their volumes.` In the future, it'll be one less thing to have to update and it doesn't particularly convey any useful information (if gp3 is not supported in a specific zone, its already implied you'll need to use a different volume type).

/lgtm